frontend/src/types/chat_message.d.ts

```diff
@@ -0,0 +1,7 @@
+type ChatMessage = {
```

Comment:
Just curious if we need to keep track of when the message is sent?

Reply:
Oops forgot about this! I can't think of a reason right now for why it might be necessary, but could be nice to have for the UX! Maybe I'll add it in?

Reply:
>  I can't think of a reason right now for why it might be necessary

Haha me too. But is just very common to have chat message having a sent time, for example like the PR comment that we are sending right now. Whether to use it or not is totally fine, I will leave the decision to you.